packages/vue/test/router.test.ts

```diff
@@ -169,6 +169,68 @@ describe('vueRouterInstrumentation()', () => {
     },
   );
 
+  it('allows to configure routeLabel=path', () => {
+    // create instrumentation
+    const instrument = vueRouterInstrumentation(mockVueRouter, { routeLabel: 'path' });
+
+    // instrument
+    instrument(mockStartTransaction, true, true);
+
+    // check
+    const beforeEachCallback = mockVueRouter.beforeEach.mock.calls[0][0];
+
+    const from = testRoutes.normalRoute1;
+    const to = testRoutes.namedRoute;
+    beforeEachCallback(to, from, mockNext);
+
+    // first startTx call happens when the instrumentation is initialized (for pageloads)
+    expect(mockStartTransaction).toHaveBeenLastCalledWith({
+      name: '/login',
+      metadata: {
+        source: 'route',
```

Comment:
Is this actually 'route' or shouldn't this be 'url' now? This has an impact on dynamic sampling and the performance product.

Reply:
This just disables using the `name`, so it uses the parametrized route when available and falls back to the URL if not.

See: https://github.com/getsentry/sentry-javascript/blob/ab14464bfc2b835496ea1f0a4cf2d05da636a816/packages/vue/src/router.ts#L98